libr/core/cmd.c

```diff
@@ -1616,6 +1623,7 @@ static int cmd_kuery(void *data, const char *input) {
 		}
 		break;
 	case '?':
+	default:
```

Comment:
if you catch defulat here, you should add an empty `case ' '`. otherwise the `k ` commands will fail

Reply:
An empty case is already being used here above. Are you talking about moving it?

Reply:
No. im talking about keeping this `defualt:` in here, but add:
```c
case ' ':
   break;
case '?':
default:
```

Reply:
That results in an error, here:
```
cmd.c:1626:2: error: duplicate case value
 1626 |  case ' ':
      |  ^~~~
cmd.c:1499:2: note: previously used here
 1499 |  case ' ':
```
That's why I was asking before.

Reply:
uhm.. then remove the default case because its interfering with other usages. see the failing tests.

Reply:
```char *s = pj_drain(pj);
r_cons_printf ("%s%c", s, 10); ```

This is what's causing the warnings, imho

Reply:
This line is not even comited. And no, the warnings are not pointing to this line. Its pretty clear wheres the error by having a duplicated case. Also. The problem are the failing tests caused by the default case that must be removed